spec/services/bulk_upload/sales/year2022/row_parser_spec.rb

```diff
@@ -107,14 +107,6 @@
     }
   end
 
-  around do |example|
-    FormHandler.instance.use_real_forms!
-
-    example.run
-
-    FormHandler.instance.use_fake_forms!
-  end
-
```

Comment:
use_fake_forms! loads json forms from spec/fixtures rather than config/forms
There are no sales forms in either directory, so this isn't making any difference in this case I don't believe.